virttest/tests/unattended_install.py

```diff
@@ -1282,6 +1283,17 @@ def copy_images():
                         raise exceptions.TestFail(install_error_exception_str)
                     if post_finish_str_found:
                         break
+                # Bug `reboot` param from the kickstart is not actually restarts
+                # the VM instead it shutsoff this is temporary workaround
+                # for the test to proceed
+                if "reboot" in open(unattended_install_config.unattended_file).read():
+                    try:
```

Comment:
below `if kickstart_reboot_bug and not vm.is_alive()` and `try` statement can be interchanged...

Reply:
taken care, Thank you.